### PR TITLE
Adds the Pestran Godhelm

### PR DESCRIPTION
**Pull request by SELFHELL**
## About The Pull Request

Adds a God-Helmet for Prestan Templars/Paladins, credit to 
@WWWWW-Git for the sprites

## Why It's Good For The Game

Having nice things is good, only one more helmet to go

good shit

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: code/modules/jobs/job_types/roguetown/adventurer/types/combat/donator/templar.dm [hunk 1]
```diff
@@ -34,11 +34,11 @@
 			head = /obj/item/clothing/head/roguetown/helmet/heavy/nochelm
 			wrists = /obj/item/clothing/neck/roguetown/psycross/noc
 		if("Pestra")
-			head = /obj/item/clothing/head/roguetown/helmet/heavy/bucket/gold // Placeholder
+			head = /obj/item/clothing/head/roguetown/helmet/heavy/pestrahelm // no more placeholder
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/roguetown/adventurer/types/combat/donator/templar.dm:37]: Remove the // no more placeholder :)

**Comment by BadAtThisGame302** [comment]
image
You're deleting all of these for some reason.

**Comment by SELFHELL** [comment]
wait what. fuck

**Comment by SELFHELL** [comment]
did my stuff conflict?

**Comment by SELFHELL** [comment]
alright icons fixed I think

**Comment by SELFHELL** [comment]
Ok third times the charm, fixed

File: code/modules/clothing/rogueclothes/hats.dm [hunk 2]
```diff
@@ -859,6 +859,16 @@
 	flags_inv = HIDEEARS|HIDEFACE|HIDEHAIR
 	prevent_crits = ALL_EXCEPT_BLUNT
 
+
+//............... Pestra Helmet ............... //
+/obj/item/clothing/head/roguetown/helmet/heavy/pestrahelm
+	name = "pestran helmet"
+	desc = "A greathelm made of rough, tainted iron. Modeled after a Plauged Carrion, a most blessed abomination of Pestra"
```
Comment (BadAtThisGame302) [inline code/modules/clothing/rogueclothes/hats.dm:866]: @Optimism333 Should probably write a description for this.

**Comment by sanshoom** [comment]
> image You're deleting all of these for some reason. Please check the IconDiffBot to see what icons you WANT to change and are ACTUALLY changing.

these are craftable now?

**Review by Optimism333 — COMMENTED** [review]
Will drop a description suggestion in a moment.

File: code/modules/clothing/rogueclothes/hats.dm (hunk 2 above)
Comment (Optimism333) [reply code/modules/clothing/rogueclothes/hats.dm:866]: Here's my suggestion, keeps the spirit of the description with some minor alterations:

"A great helmet made of coarse, tainted steel. It is modeled after a plagued carrion, a blessed abomination of Pestra."

**Review by Optimism333 — COMMENTED** [review]
If you could resolve conflict please, this can go in immediately after.

**Comment by SELFHELL** [comment]
> If you could resolve conflict please, this can go in immediately after.

Done

**Review by Optimism333 — APPROVED** [review]

**Pull request merged**